cassandra-reaper.yaml

```diff
@@ -46,7 +46,8 @@ pipeline:
       source /usr/share/nvm/nvm.sh
       nvm install v12
       nvm use v12
-      mvn clean package -DskipTests
+      MAVEN_OPTS="-Xmx384m" mvn -B install -DskipTests -Drat.skip=true
+      # mvn clean package -DskipTests
```

Comment:
Can we just remove this line? Do we expect to need it back soon?

Reply:
let me remove it

Reply:
What does the `-Drat.skip=true` argument do, out of interest? Looking at their CI scripts I see the rest of the arguments, but not that one:
- https://github.com/thelastpickle/cassandra-reaper/blob/da5096f71cd25558bd60e26e830c29e78d0e1160/.github/workflows/ci.yaml#L522

Reply:
I'm not sure but the build was failing due to rat plugin check, it think its something like compliance checking : https://creadur.apache.org/rat/apache-rat-plugin/